packages/relay/test/comment.test.ts

```diff
@@ -107,7 +115,32 @@ describe('COMMENT /comment', function () {
                 return res.body.txHash
             })
 
-        await ethers.provider.waitForTransaction(txHash)
+        // to check if the event emitting on chain correctly
+        // Post Comment = {
+        //  type in solidity        type in typescript
+        //  uint256 epochKey, [0]   BigInt
+        //  uint256 postId,   [1]   BigInt
+        //  uint256 commentId [2]   BigInt
+        //  uint256 epoch     [3]   BigInt
+        //  string cid        [4]   String
+        // }
+        const epk = epochKeyProof.epochKey as bigint
+        const receipt = await ethers.provider.waitForTransaction(txHash)
+        const unirepAppInterface = new ethers.utils.Interface(abi)
+        const rawEvent = receipt.logs
+            .map((log) => unirepAppInterface.parseLog(log))
+            .find((log) => log.name == 'Comment')
+        const commentEvent = rawEvent?.args
+
+        expect(commentEvent).to.not.be.undefined
+        if (commentEvent) {
+            expect(commentEvent[0].toString()).equal(epk.toString())
+            expect(commentEvent[1]).equal('0')
```

Comment:
Recommend to fetch latestPostId before creating post and use that as the postId, and same for the other emit variables. Hard code will make it harder if we want to adjust the test or test tools in the future.